src/app.tsx

```diff
@@ -110,13 +110,15 @@ export const Application = () => {
         [currentPath, sel]
     );
 
+    const visibleFiles = useMemo(() => {
+        return !showHidden
+            ? files.filter(file => !file.name.startsWith("."))
+            : files;
+    }, [files, showHidden]);
+
```

Comment:
Would be nice to know why this was needed? Or a short comment that this saves selection a new item not re-filter files passed to the FolderView.

Reply:
Thanks! I have follow up plans to just move this into NavigatorCard or however it is called, it's only used there so makes no sense to have here.